app/i18n/kr/sub.php

```diff
@@ -2,8 +2,8 @@
 
 return array(
 	'api' => array(
-		'documentation' => 'Copy the following URL to use it within an external tool.',// TODO
-		'title' => 'API',// TODO
+		'documentation' => '외부 도구에서 API를 사용하기 위해서 아래 URL을 사용하세요.',
+		'title' => 'API',
```

Comment:
@gsongsong  As the translation is the same as in english. You should add exceptions for the validation tool.
Check in ```cli/i18n/ignore``` for examples. All keys added in those files are discarded from the completion validation tool.

Reply:
Thank you. I fixed it. But validation will also fail due to `mute`. I need to know the exact behavior of it before translation.